Altis_Life.Altis/stringtable.xml

```diff
@@ -887,7 +887,7 @@
             <Polish>Nie masz działalność z wykorzystaniem tego produktu.</Polish>
             <Portuguese>Você não tem nenhum negócio usando este.</Portuguese>
             <Russian>У вас нет бизнеса с помощью этого.</Russian>
-            <German>Sie haben kein Geschäft mit diesem.</German>
+            <German>Du hast kein Zugriff um das zu benutzen.</German>
```

Comment:
Yours isn't wrong, but I think that sounds better: `Du hast hier nichts verloren!`